crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -415,7 +436,8 @@ impl Entities {
     pub fn get(&self, entity: Entity) -> Option<EntityLocation> {
         if (entity.id as usize) < self.meta.len() {
             let meta = &self.meta[entity.id as usize];
-            if meta.generation != entity.generation
+            if meta.generation.is_none()
+                || meta.generation.unwrap() != entity.generation
```

Comment:
```suggestion
            if meta.generation != Some(entity.generation)
```
or
```suggestion
            if meta.generation? != entity.generation
```
would both also work

Reply:
Whoops, must have been a little tired when I went through this, did it "correct" already elsewhere